docs/dev.md

```diff
@@ -20,6 +20,17 @@ export KAPPCTRL_E2E_NAMESPACE=kappctrl-test
 ./hack/test-all.sh
 ```
 
+### Known Issues
```

Comment:
i would rename that to "troubleshooting tips"

Reply:
Makes sense, I think I had it as that and then my brain told me it was closer to a known issue than a troubleshooting. But I agree, I like troubleshooting more. So lets make it that.